pkg/openwsn/contrib/openwsn.c

```diff
@@ -52,46 +61,42 @@ kernel_pid_t openwsn_get_pid(void)
     return _pid;
 }
 
-#ifdef MODULE_OPENWSN_RADIO
-void openwsn_set_addr_16b(netdev_t* dev)
+void* _radio_init_dev(void)
 {
-    uint8_t addr[IEEE802154_SHORT_ADDRESS_LEN];
-    dev->driver->get(dev, NETOPT_ADDRESS, addr, IEEE802154_SHORT_ADDRESS_LEN);
-    open_addr_t id;
-    id.type = ADDR_16B;
-    memcpy(&id.addr_16b, addr, IEEE802154_SHORT_ADDRESS_LEN);
-    idmanager_setMyID(&id);
-}
+    void* dev = NULL;
+    /* avoid cppcheck style (redundantAssignment)*/
+    (void) dev;
+#ifdef MODULE_OPENWSN_RADIO_NETDEV
```

Comment:
Here and below
```suggestion
#if IS_USED(MODULE_OPENWSN_RADIO_NETDEV)
```

Reply:
Can't do this, it will complain about `at86rf2xx_dev` not existing since that one is guarded by an `ifdef` (and I don't think I can do it differently without compiler complaints)